src/webui/modules/home/index.js

```diff
@@ -98,11 +102,16 @@ export default class Home extends React.Component {
   }
 
   render() {
+    const {packages, loading} = this.state;
     return (
-      <div>
+      <Fragment>
```

Comment:
Do we have support for shorthand fragment syntax? (`<> </>`)

Reply:
We have support for shorthand.